### PR TITLE
chore: bump snapshot version to 24.5

### PR DESCRIPTION
**Pull request by platosha**

### CODE REVIEW

File: .github/workflows/validation.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ defaults:
 
 on:
   push:
-    branches: [ main, '2.5', '2.4', '2.3', '2.2', '1.4', '1.3' ]
+    branches: [ main, '24.4', '2.5', '1.4', '1.3' ]
```
Comment (taefi) [inline .github/workflows/validation.yml:9]: I wonder if we should add `1.5` here and remove `1.3` already.

File: packages/java/tests/pom.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
     <packaging>pom</packaging>
 
     <properties>
-        <testbench.element.version>24.4-SNAPSHOT</testbench.element.version>
+        <testbench.element.version>24.5-SNAPSHOT</testbench.element.version>
```
Comment (taefi) [inline packages/java/tests/pom.xml:18]: Could we postpone upgrading `testbench.element` to the near future when it's released? Now, it is breaking the build.

File: pom.xml [hunk 3]
```diff
@@ -75,7 +75,7 @@
     <jackson.version>2.17.0</jackson.version>
     <junit.version>5.10.1</junit.version>
     <mockito.version>4.5.0</mockito.version>
-    <flow.version>24.4-SNAPSHOT</flow.version>
+    <flow.version>24.5-SNAPSHOT</flow.version>
```
Comment (taefi) [inline pom.xml:78]: Could we postpone upgrading `Flow` to the near future when it's released? Now, it is breaking the build.

**Review by taefi — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (platosha) [reply pom.xml:78]: Seems to work now.

**Review by taefi — APPROVED** [review]

**Pull request merged**